delphi-frontend/src/main/java/au/com/integradev/delphi/antlr/ast/node/AttributeNodeImpl.java

```diff
@@ -56,7 +63,11 @@ public NameOccurrence getConstructorNameOccurrence() {
 
   @Override
   public ArgumentListNode getArgumentList() {
-    return (ArgumentListNode) getChild(1);
+    DelphiNode node = getChild(isAssembly() ? 3 : 1);
```

Comment:
This also seems like it should be `isAssembly() ? 2 : 1`.

Reply:
Ha! This one wouldn't have made sense with or without the colons in the tree. Weird.